<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Other</class>
 <widget class="QDialog" name="Other">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color:rgb(255, 255, 255); 
}

QDialog{
background:url(://image/backGround.png)
}</string>
  </property>
  <widget class="QLabel" name="label_power">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Please wait while shutdown..</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Thank">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Thank you for your use!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_process">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>256</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
